STIPS/STIP-009.md

```diff
@@ -0,0 +1,1195 @@
+
+##### FeeState
+
+| Type 	| Name 	| Description 	|
+|------	|------	|-------------	|
+|address|feeRecipient|Address to accrue fees to|
+|uint256|maxIssueFee|Max issuance fee manager commits to using (1% = 1e16, 100% = 1e18)|
+|uint256|issueFee|Percent of Set accruing to manager on issuance (1% = 1e16, 100% = 1e18)|
+|uint256|maxRedeemFee|Max redemption fee manager commits to using (1% = 1e16, 100% = 1e18)|
+|uint256|redeemFee|Percent of Set accruing to manager on redemption (1% = 1e16, 100% = 1e18)|
+
+#### Global Variables
+
+| Type 	| Name 	| Description 	|
+|------	|------	|-------------	|
+|IIssuanceModule|issuanceModule|Issuance Module for Set Token|
+
+#### Public Variables
+
+| Type 	| Name 	| Description 	|
+|------	|------	|-------------	|
+|mapping(address => IDelegatedManager)|setManagers|Mapping from Set Token to DelegatedManager|
+
+#### Functions
+
+| Name  | Caller  | Description     |
+|------	|------	|-------------	|
+|initializeExtension|owner|Initialize the BasicIssuanceExtension on the DelegatedManager |
+|initializeExtensionAndModule|owner|Initialize the BasicIssuanceModule on the SetToken and the BasicIssuanceExtension on the DelegatedManager |
+|updateIssueFee|owner|Update issue fee on IssuanceModule|
+|updateRedeemFee|owner|Update redeem fee on IssuanceModule|
+
+----
+
+### Functions
+
+> initializeExtension
+
+Initialize the BasicIssuanceExtension on the DelegatedManager
+
+```solidity
+function initializeExtension(address _delegatedManager) external
+```
++ require that *msg.sender* is the *_delegatedManager.owner*
++ require that extension state in delegatedManager's *extensionAllowlist* is *PENDING*
++ set *setManagers[_setTokenAddress]* to manager
++ call `manager.initializeExtension()`
++ emit *ExtensionInitialized* event
+
+----
+
+> initializeModuleAndExtension
+
+Initialize the BasicIssuanceModule and the BasicIssuanceExtension. (Can only be called by delegatedManager owner, checked in `initializeExtension`)
+
+```
+function initializeModuleAndExtension(address _delegatedManager, IManagerIssuanceHook _preIssueHook) external
+```
+
++ call *initializeExtension(_delegatedManager)*
++ Formulate call to initialize module from manager
+    ```solidity
+    bytes memory callData = abi.encodeWithSelector(
+        IBasicIssuanceModule.initialize.selector,
+        delgatedManager.setToken(),
+        _preIssueHook
+    );
+    ```
+
++ call *invokeManager(basicIssuanceModule, callData)*
+
+### StreamingFeeSplitExtension
+
+#### Inheritance
+
+- BaseGlobalExtension
+
+#### Structs
+
+##### FeeState
+
+| Type 	| Name 	| Description 	|
+|------	|------	|-------------	|
+|address|feeRecipient|Address to accrue fees to|
+|uint256|maxStreamingFeePercentage|Max streaming fee manager commits to using (1% = 1e16, 100% = 1e18)|
+|uint256|streamingFeePercentage|Percent of Set accruing to manager annually (1% = 1e16, 100% = 1e18)|
+|uint256|lastStreamingFeeTimestamp|Timestamp last streaming fee was accrued|
```

Comment:
repeat: I think we don't actually need this. See (https://github.com/SetProtocol/set-v2-strategies/blob/b2990446f10725bed68faac11f6d197b86885cec/contracts/extensions/StreamingFeeSplitExtension.sol#L130)